js/modules/k6/crypto/crypto_test.go

```diff
@@ -252,3 +252,48 @@ func TestOutputEncoding(t *testing.T) {
 		assert.EqualError(t, err, "GoError: Invalid output encoding: someInvalidEncoding")
 	})
 }
+
+func TestHMac(t *testing.T) {
+	if testing.Short() {
+		return
+	}
+
+	rt := goja.New()
+	rt.SetFieldNameMapper(common.FieldNameMapper{})
+
+	root, _ := lib.NewGroup("", nil)
+	state := &common.State{Group: root}
+
+	ctx := context.Background()
+	ctx = common.WithState(ctx, state)
+	ctx = common.WithRuntime(ctx, rt)
+
+	rt.Set("crypto", common.Bind(rt, &Crypto{}, &ctx))
+
+	t.Run("Valid", func(t *testing.T) {
```

Comment:
I wrote a default case in createHmac method for invalid or not supported algorithms that throw an "Invalid algorithm error", and I added it to unit tests, I think createHash must have the same default case, because trying to update or digest on the hasher created will cause a runtime error with nil pointer dereference, because hasher.hash is never asigned.